src/chains/ethereum/ethereum/src/forking/cache.ts

```diff
@@ -1,19 +1,18 @@
-import { Account, Address as EJS_Address } from "ethereumjs-util";
+import { Account, Address } from "ethereumjs-util";
 import Cache from "@ethereumjs/vm/dist/state/cache";
 import { GanacheTrie } from "../helpers/trie";
 import { ForkTrie } from "./trie";
 
 export class ForkCache extends Cache {
   constructor(trie: GanacheTrie) {
-    super(trie);
+    /**
+     * Looks up address in underlying trie.
+     * @param address - Address of account
+     */
+    const lookupAccount = async (address: Address) => {
+      const rlp = await (trie as ForkTrie).get(address.buf);
+      return rlp ? Account.fromRlpSerializedAccount(rlp) : new Account();
+    }
+    super({ getCb: lookupAccount, putCb: trie.put.bind(trie), deleteCb: trie.del.bind(trie) });
```

Comment:
This is the real change. The @ethereumjs/vm `Cache` no longer accepts a Merkle Patricia Trie instance, but instead takes the options listed here. It also replaces the `_lookupAccount` function internally with the `getCb` option, which is why it was removed below.

I don't know what these have a "Cb" prefix and don't know what `Cb` means in this context. I can tell you that these are _not_ "callbacks" though, they actually fetch the data and return Promises.
.